### DISCUSSION TITLE
New as a contributor - I'm trying to find a starter issue to work on?

### DISCUSSION THREAD

**Discussion by mcmho** [open-telemetry/opentelemetry-java-instrumentation#3887] [Q&A]
Hi,  I have been working couple years on java agent for my company.  I would like to get started contributing to the OT OSS community.   I am looking for a starter issue to get myself familiar with the OSS PR process.   

I tried to look here - https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues 
anyone has any suggestion?  😃 

Thanks,
Matthew

**Comment by trask** [comment]
hi @mcmho! check out #2713, I think it's an excellent place to start because you can look at examples of instrumentations that have already been converted to see what's involved, and it will introduce you to the instrumentation guts, which can be a lot also, but feel free to reach out over slack for help any time!

**Reply by mcmho** [reply]
Thanks @trask for your reply!  I can definitely help and dig deep into the instrumentation guts.   
I took a look at #2713.  It seems pretty involving though.  

as a first commit, I was wondering if there is a simpler one involving refactoring and such.

**Reply by trask** [reply]
I just did a light issue scrub, and these looks like some more options:

* #2958
* #2674
* #2481 (the spring boot starters in general could probably use some ❤️)
* #2445
* #1708
